sentry-sample/src/main/AndroidManifest.xml

```diff
@@ -3,88 +3,100 @@
 
-    <!--    how to disable the NDK-->
-    <!--    <meta-data android:name="io.sentry.ndk.enable" android:value="false" />-->
+        <!--    how to disable the NDK-->
+        <!--    <meta-data android:name="io.sentry.ndk.enable" android:value="false" />-->
 
-    <!--    how to set a custom release-->
-    <!--    <meta-data android:name="io.sentry.release" android:value="io.sentry.sample@1.0.0+10001" />-->
+        <!--    how to set a custom release-->
+        <!--    <meta-data android:name="io.sentry.release" android:value="io.sentry.sample@1.0.0+10001" />-->
 
-    <!--    how to enable automatic session tracking-->
-    <meta-data android:name="io.sentry.session-tracking.enable" android:value="true" />
+        <!--    how to enable automatic session tracking-->
+        <!--    <meta-data android:name="io.sentry.session-tracking.enable" android:value="true" />-->
 
-    <!--    how to set an environment-->
-    <!--    <meta-data android:name="io.sentry.environment" android:value="debug" />-->
+        <!--    how to set an environment-->
+        <!--    <meta-data android:name="io.sentry.environment" android:value="debug" />-->
 
-    <!--    how to change the session tracking interval-->
-    <meta-data android:name="io.sentry.session-tracking.timeout-interval-millis" android:value="10000" />
+        <!--    how to change the session tracking interval-->
+        <meta-data
+            android:name="io.sentry.session-tracking.timeout-interval-millis"
+            android:value="10000" />
 
-    <!--    how to enable and set a sampleRate (anything between 0.01 and 1.0), it's disabled by default-->
-    <!--    <meta-data android:name="io.sentry.sample-rate" android:value="0.5" />-->
+        <!--    how to enable and set a sampleRate (anything between 0.01 and 1.0), it's disabled by default-->
+        <!--    <meta-data android:name="io.sentry.sample-rate" android:value="0.5" />-->
 
-    <!--    how to disable default sentry uncaught exception-->
-    <!--    <meta-data android:name="io.sentry.uncaught-exception-handler.enable" android:value="false" />-->
+        <!--    how to disable default sentry uncaught exception-->
+        <!--    <meta-data android:name="io.sentry.uncaught-exception-handler.enable" android:value="false" />-->
 
-    <!--    how to disable the activity lifecycle breadcrumbs integration-->
-    <!--    <meta-data android:name="io.sentry.breadcrumbs.activity-lifecycle" android:value="false" />-->
+        <!--    how to disable the activity lifecycle breadcrumbs integration-->
+        <!--    <meta-data android:name="io.sentry.breadcrumbs.activity-lifecycle" android:value="false" />-->
 
-    <!--    how to disable the app lifecycle breadcrumbs integration-->
-    <!--    <meta-data android:name="io.sentry.breadcrumbs.app-lifecycle" android:value="false" />-->
+        <!--    how to disable the app lifecycle breadcrumbs integration-->
```

Comment:
this is just a linter thing? I dunno what I'm reviewing here

Reply:
probably changed by the IDE automatically and will be reverted, Draft PR